src/fallback.rs

```diff
@@ -919,7 +919,7 @@ named!(group -> Group, alt!(
 ));
 
 fn symbol_leading_ws(input: Cursor) -> PResult<TokenTree> {
```

Comment:
This function should be deleted if it no longer does anything different from `symbol`.